src/test/regress/sql/background_rebalance_parallel.sql

```diff
@@ -204,11 +204,9 @@ SELECT citus_rebalance_start AS job_id from citus_rebalance_start() \gset
 -- see dependent tasks to understand which tasks remain runnable because of
 -- citus.max_background_task_executors_per_node
 -- and which tasks are actually blocked from colocation group dependencies
-SELECT D.task_id,
-       (SELECT T.command FROM pg_dist_background_task T WHERE T.task_id = D.task_id),
-       D.depends_on,
-       (SELECT T.command FROM pg_dist_background_task T WHERE T.task_id = D.depends_on)
-FROM pg_dist_background_task_depend D  WHERE job_id in (:job_id) ORDER BY D.task_id, D.depends_on ASC;
+SELECT (SELECT T.command FROM pg_dist_background_task T WHERE T.task_id = D.task_id),
+       (SELECT T.command depends_on_command FROM pg_dist_background_task T WHERE T.task_id = D.depends_on)
+FROM pg_dist_background_task_depend D  WHERE job_id in (:job_id) ORDER BY 1, 2 ASC;
```

Comment:
Can we add another query that shows the task ids as I'd like to clarify those for the following tests 

``` SQL
SELECT task_id,
       depends_on
FROM pg_dist_background_task_depend WHERE job_id in (:job_id) ORDER BY 1, 2 ASC;
```